### ISSUE TITLE
[Victory] The save model button should be disabled.

### ISSUE THREAD

**Issue by shub1989** [eMoflon/emoflon-ibex-ui#165]
The save model should save current state of the models once, and should be disabled later.

**Comment by LegionaryCohort** [comment]
Would it perhaps be even more useful to open a save dialog instead and allow the user to save to a location of their choosing?

**Comment by anthonyanjorin** [comment]
Yes, but please with /instances and time stamp as default location and suffix, so I can just click ok if I’m happy with that.
